Formula/git-game.rb

```diff
@@ -1,14 +1,16 @@
 class GitGame < Formula
   desc "Game for git to guess who made which commit"
   homepage "https://github.com/jsomers/git-game"
-  url "https://github.com/jsomers/git-game/archive/1.2.tar.gz"
-  sha256 "d893b2c813388754c16d867cf37726cd7e73c9ccd316735aac43bf1cd3ab1412"
+  url "https://github.com/coffee-dan/git-game/archive/1.3.tar.gz"
+  sha256 "439eb7cdb9791e2d96b05e44e4e2723db60fca11afe919c592291b6fa2c86163"
   license "MIT"
 
   bottle do
     sha256 cellar: :any_skip_relocation, all: "c700a65b591b1e95ab45bcd0349a75aeb8fdc9915480c7323a34830cd27075fd"
   end
 
+  depends_on "ruby"
+
   def install
     bin.install "git-game"
```

Comment:
The shebang here probably needs to be `/usr/bin/env ruby` instead of `/usr/bin/ruby`.

Reply:
Thank you so much! I will test that out now.